src/reducers/RoomSlice.js

```diff
@@ -70,9 +73,22 @@ const roomSlice = createSlice({
     isStrict: false, 
     userIsHost: false,
     userInRoom: false,
-    roomUserName: null
+    userInVestibule: false,
+    roomUserName: null,
+    roomPassword: null,
+    roomJoinRequestAccepted: false,
```

Comment:
`roomJoinRequestAccepted` is fine to add, we could discuss later what this is supposed to represent for host vs non-host